helm_deploy/values-dev.yaml

```diff
@@ -78,4 +70,4 @@ secrets:
   appStore: https://laa-crime-application-store-dev.apps.live.cloud-platform.service.justice.gov.uk
 
 service_account:
-  name: laa-claim-non-standard-magistrate-fee-dev-irsa
\ No newline at end of file
+  name: laa-claim-non-standard-magistrate-fee-dev-irsa
```

Comment:
I think we need a way of providing dynamic dev environments with their own DB. Otherwise a dodgy migration in one PR will break every other environment. In CCQ we used a postgres pod with a persistent volume claim. I guess you could also use RDS, creating a new database with a custom name inside the dev RDS instance for each branch and then deleting it on cleanup?